src/dbnode/storage/index/convert/convert_test.go

```diff
@@ -171,6 +172,113 @@ func TestFromSeriesIDAndTagIterReuseBytesFromSeriesId(t *testing.T) {
 	}
 }
 
+func TestFromSeriesIDAndEncodedTagsInvalid(t *testing.T) {
```

Comment:
Nit: why not merge it into `TestFromSeriesIDAndEncodedTagsInvalidEncodedTags`? Initially I thought this was the only "invalid" test case that we have.

Reply:
https://github.com/m3db/m3/pull/3087/commits/a5436e8235c627486c9b0aec6ae438a82e5c30c7#diff-4b7b04cd8028a198a21da071cd0a8bbcc9d2f7fd0f6f4bd896a73019eccdfdfeR223-R238